kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/producer/DefaultProducerFactoryIntegrationTest.java

```diff
@@ -104,36 +101,33 @@ void testDefaultConfirmationModeForTransactionalProducer() {
 
     @Test
     void testConfiguringInvalidCacheSize() {
-        DefaultProducerFactory.Builder<Object, Object> builder = builder().configuration(minimal(getBootstrapServers()))
-                                                                          .producerCacheSize(-1);
-        assertThrows(AxonConfigurationException.class, builder::build);
+        DefaultProducerFactory.Builder<Object, Object> builder = builder().configuration(minimal(getBootstrapServers()));
```

Comment:
These are fixes expected from Sonar, right?

Reply:
Well, kind of, exidently broke the integration test before. Now it's clearer what's been tested. Right now on master the integration test is failing.

Reply:
Cool, thanks for the explanation here.